### PR TITLE
Add a Go example to readme

### PR DESCRIPTION
**Pull request by brianfoshee**

### CODE REVIEW

**Comment by dmathieu** [comment]
Thank you for this, it's awesome! 🌷 
A couple thoughts.

I think the examples should remain as simple as possible. The main function here could be misleading. How about just having an HandlerFunc with an anonymous function inside it?

I'm personnally not a big fan of methods with several return paths. How about defining a local variable with the key we need to print, and do the Fprintf only once at the end if it's set?

Finally, logging is kind of too opinionated for an example (I would never log phrases like this. Logs need to be searchable). Could you remove them?

**Comment by brianfoshee** [comment]
The example has been updated accordingly :)

File: README.md [hunk 1]
```diff
@@ -119,6 +118,46 @@ end
...
 ```
 
+### Go apps
+
+Add the following handler to your app:
+
+```go
+http.HandleFunc("/.well-known/acme-challenge/", func(w http.ResponseWriter, r *http.Request) {
+  pt := strings.TrimPrefix(r.URL.Path, "/.well-known/acme-challenge/")
+  rk := ""
```
Comment (dmathieu) [inline README.md:128]: This is a nitpick, but the Go convention when setting vars which aren't used right away is to use `var rk string`.

**Comment by dmathieu** [comment]
Thank you Brian!

**Pull request merged**